samples/bluetooth/broadcast_config_tool/README.rst

```diff
@@ -0,0 +1,677 @@
+=======================================
+
+The command sets the number of subgroups for a BIG.
+
+The maximum number of subgroups for each BIG is set by ``CONFIG_BT_BAP_BROADCAST_SRC_SUBGROUP_COUNT``.
+
+Usage:
+  .. parsed-literal::
+     :class: highlight
+
+     bct num_subgroups <number> <BIG index>
+
+For example:
+  .. parsed-literal::
+     :class: highlight
+
+     bct num_subgroups 2 0
+
+This will set the number of subgroups for BIG0 to 2.
+
+num_bises - Set number of BISes
+===============================
+
+The command sets the number of BISes for a given subgroup.
+
+The maximum number of BISes for each subgroup is set by ``CONFIG_BT_BAP_BROADCAST_SRC_STREAM_COUNT``.
+
+Usage:
+  .. parsed-literal::
+     :class: highlight
+
+     bct num_bises <number> <BIG index> <Subgroup index>
+
+For example:
+  .. parsed-literal::
+     :class: highlight
+
+     bct num_bises 2 0 0
+
+This will set the number of BISes for subgroup 0 in BIG0 to 2.
+
+context - Set context
+=====================
+
+The command sets the context metadata for a subgroup.
+
+Usage:
+  .. parsed-literal::
+     :class: highlight
+
+     bct context <context> <BIG index> <Subgroup index>
+
+The supported contexts can be shown be typing:
+
+  .. parsed-literal::
+     :class: highlight
+
+     bct context print
+
+For example:
+  .. parsed-literal::
+     :class: highlight
+
+     bct context Media 0 0
+
+This will set the context for subgroup 0 in BIG0 to Media.
+
+location - Set location
+=======================
+
+The command sets the location metadata for a BIS.
+
+Usage:
+  .. parsed-literal::
+     :class: highlight
+
+     bct location <location> <BIG index> <Subgroup index> <BIS index>
+
+The supported locations can be shown be typing:
+  .. parsed-literal::
+     :class: highlight
+
+     bct location print
+
+For example:
+  .. parsed-literal::
+     :class: highlight
+
+     bct location fl 0 0 0
+
+This will set the location for BIS 0 in subgroup 0 in BIG0 to Front Left.
+
+broadcast_name - Set broadcast name
+===================================
+
+The command sets the broadcast name metadata for a BIG.
+
+Usage:
+  .. parsed-literal::
+     :class: highlight
+
+     bct broadcast_name <name> <BIG index>
+
+For example:
+  .. parsed-literal::
+     :class: highlight
+
+     bct broadcast_name Lecture 0
+
+This will set the broadcast name for BIG0 to "Lecture".
+
+For longer names with spaces, use " ".
+
+For example:
+  .. parsed-literal::
+     :class: highlight
+
+     bct broadcast_name "Lecture hall" 0
+
+
+encrypt - Set broadcast code
+============================
+
+The command enables or disables encryption for a given BIG.
+
+Usage:
+  .. parsed-literal::
+     :class: highlight
+
+     bct encrypt <0/1> <BIG index> [broadcast_code]
+
+The broadcast code is a 16-character string that is used to encrypt the broadcast, but shorter codes are also possible.
+
+For example:
+  .. parsed-literal::
+     :class: highlight
+
+     bct encrypt 1 0 Auratest
```

Comment:
So the first "1" is to enable ?  That is not fully clear.

Reply:
The documentation says to use either 0 or 1 as the first argument, and then I included an example for enabling and one example for disabling. How would you make it more clear?